### DISCUSSION TITLE
My live video in HyperHDR is showing up pink

### DISCUSSION THREAD

**Discussion by Rilf10** [awawa-dev/HyperHDR#539] [Q&A]
Hi,
I just set up HyperHDR using a raspberrypi 3B and WB2812B lights with a common 5V 10A power supply
The capturecard(4k60hz) is inputting from a firestick 4k to the TV
the leds are showing up as a pink hue and the live video from the HyperHDR link is showing up as a distorted pink image of the one in the TV.
Please help me resolve these issues

Thank you very much for your help

**Comment by awawa-dev** [comment]
Hi
Probably you try to capture Dolby Vision stream that is not supported by the grabber.

**Reply by Rilf10** [reply]
Oh, How do I rectify this?
Do I turn it off from the Firestick itself?

**Reply by Rilf10** [reply]
do you also reckon there could be something wrong with the HDMI cable or the capture card as well?

**Reply by awawa-dev** [reply]
I dont use Firestick...maybe there is an option to use HDR10 instead of DV in the settings?

**Reply by Rilf10** [reply]
Yup, that fixed it for me...thank you very much!!

**Reply by xSterney** [reply]
how did you change the setting DV to HDR10?

**Reply by AstaRom** [reply]
> how did you change the setting DV to HDR10?

- [reddit.com](https://www.reddit.com/r/fireTV/comments/z1l16v/comment/ixbna1c/?utm_source=share&utm_medium=web3x)

**Reply by Nightreaver** [reply]
Any known grabber with DV support ?
